docs/io/ascii/write.rst

```diff
@@ -231,3 +231,207 @@ details.
+   93-107  F15.12  arcsec DEs         Declination (arcsec)                  
+  --------------------------------------------------------------------------------
+  Notes:
+  --------------------------------------------------------------------------------
+  ASASSN-15lh 2.87819e-09 0.0250        1e-10 2019.0 100 22.0  2.0 15.450000000007 -61.0 39.0 34.599996000001
+  ASASSN-14li 2.55935e-08 0.0188  2.044 4e+03 2019.0 100 12.0 48.0 15.224407200005  17.0 46.0 26.496624000004
+
+And the file ``ecliptic_cols.dat`` will look like::
+
+  Title:
+  Authors:
+  Table:
+  ================================================================================
+  Byte-by-byte Description of file: table.dat
+  --------------------------------------------------------------------------------
+   Bytes Format Units  Label     Explanations
+  --------------------------------------------------------------------------------
+    1- 11  A11     ---    Name        Description of Name                        
+   13- 23  E11.6   keV    Temperature [0.0/0.01] Description of Temperature      
+   25- 30  F6.4    10+22  nH          [0.01/0.03] Description of nH              
+   32- 37  F6.3   10+12Jy Flux        ? Description of Flux                      
+   39- 43  E5.1    mag    magnitude   [0.0/3981.08] Description of magnitude     
+   45- 50  F6.1    yr     Obs         [2019.0/2019.0] Time of Observation        
+   52- 54  I3      s      Cadence     [100] Description of Cadence               
+   56- 71  F16.12  deg    ELON        Ecliptic Longitude (geocentrictrueecliptic)
+   73- 88  F16.12  deg    ELAT        Ecliptic Latitude (geocentrictrueecliptic) 
+  --------------------------------------------------------------------------------
+  Notes:
+  --------------------------------------------------------------------------------
+  ASASSN-15lh 2.87819e-09 0.0250        1e-10 2019.0 100 306.224208650096 -45.621789850825
+  ASASSN-14li 2.55935e-08 0.0188  2.044 4e+03 2019.0 100 183.754980099243  21.051410763027
+
+Finally, MRT and CDS have some specific naming conventions for columns
+(`<https://journals.aas.org/mrt-labels/#reflab>`_). For example, if a column contains
+the mean error for the data in a column named ``label``, then this column should be named ``e_label``.
+These kind of relative column naming cannot be enforced by the CDS/MRT writer
+because it does not know what the column data means and thus, the relation between the
+columns cannot be figured out. Therefore, it is up to the user to use ``Table.rename_colums``
+to appropriately rename any columns before writing the table to MRT/CDS format.
+The following example shows a similar situation, using the option to send the output to
+``sys.stdout`` instead of a file::
+
+  >>> table['error'] = [1e4, 450] * u.Jy  # Error in the Flux values.
+  >>> outtab = table.copy()  # So that changes don't affect the original table.
+  >>> outtab.rename_column('error', 'e_flux')
```

Comment:
```suggestion
  >>> outtab.rename_column('error', 'e_Flux')
```
Should this not start with a capital letter, if the original "Flux" column also starts with a capital letter?